test/testfolderwatcher.cpp

```diff
@@ -18,7 +18,7 @@ void touch(const QString &file)
     QString cmd;
     cmd = QString("touch %1").arg(file);
     qDebug() << "Command: " << cmd;
-    system(cmd.toLocal8Bit());
+    (void)! system(cmd.toLocal8Bit());
```

Comment:
Better fix would be to actually check the return code and issue an error (assert or such) if it's different from 0. This would actually make things easier to debug if somehow this failed.